### ISSUE TITLE
List of errors for which we need informative messages 

### ISSUE THREAD

**Issue by jaspercooper** [DeclareDesign/DeclareDesign#129]
When you have "labels" instead of "label" in your estimator declaration, you get:

"Error in substitute(estimator)[-1L] : 
  object of type 'symbol' is not subsettable"

We need an error that makes it clear what the problem is here. 

More detail below. 

the individual estimators generated by the lapply() function in the template are working as intended, so the following modified design (where des was the original design) can be diagnosed without error:

des2 <- modify_design(des, estimator = des$estimator[[1]])

However, when you put these individual estimators in a list (alone or in combination), it returns the same error cited above, for example:

des3 <- modify_design(des, estimator = list(des$estimator[[1]]))

**Comment by graemeblair** [comment]
This is not correct, I think - it would return this message regardless of whether you include labels or not, this is an error when you don't provide a label (which if you provide labels is equivalent, because of ...). I will try to fix the underlying error.
